src/screens/Home/components/Login/index.tsx

```diff
@@ -1,9 +1,99 @@
-import React from 'react';
+/* eslint-disable no-negated-condition */
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-query';
+import { useHistory } from 'react-router-dom';
+import i18next from 'i18next';
+
+import WoloxLogo from 'assets/LogoWolox-Original.png';
+import { login, LoginProps } from 'services/LoginService';
+import Loading from 'components/Spinner/components/loading';
 
 import styles from './styles.module.scss';
 
+type InputsValues = {
+  email: string;
+  password: string;
+};
+
+enum UserFieldIds {
+  EMAIL = 'email',
+  PASSWORD = 'password'
+}
+
 function Login() {
-  return <div className={styles.login}>Login</div>;
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<InputsValues>();
+  const history = useHistory();
+  const [notCredentials, setNotCredentials] = useState(false);
+
+  const mutation = useMutation(login, {
+    onError: error => {
+      console.log(error);
+      setNotCredentials(true);
+    },
+    onSuccess: data => {
+      console.log(data.headers);
+      history.push('/sign_up');
+    }
+  });
+
+  const onSubmit = (user: LoginProps) => mutation.mutate(user);
+
+  return (
+    <div className="column center middle full-width">
+      <form className={`m-bottom-2 ${styles.loginForm}`} onSubmit={handleSubmit(onSubmit)}>
+        <img src={WoloxLogo} className={`m-bottom-3 ${styles.logo}`} />
```

Comment:
always try to set an alt message in case the image doesn't load. Not all images can have it, mostly the ones that represent something. In this case you could just put `alt="Wolox"`